src/sources/aws_kinesis_firehose/handlers.rs

```diff
@@ -38,16 +38,15 @@ pub async fn firehose(
     acknowledgements: bool,
     mut out: SourceSender,
 ) -> Result<impl warp::Reply, reject::Rejection> {
+    // TODO: Move this up into the caller
```

Comment:
Is this something we want to do in this PR or a follow up?

Reply:
No, I want this done here, I missed moving it before submitting.